.github/workflows/publish.yml

```diff
@@ -0,0 +1,26 @@
+name: Publish
+'on':
+  push:
+    branches:
+      - master
```

Comment:
Shouldn't this be removed after testing too?

Reply:
Wonder if there is a on tag option...

Reply:
https://github.com/omerbenamram/pyevtx-rs/pull/13/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR5-R7